cogip/tools/camera/__init__.py

```diff
@@ -0,0 +1,8 @@
+import logging
```

Comment:
In commit 4c9a64690d0292c4b30007ed5d9e89d12daa6973 message, quotes around `info` are different.